packages/styles/forms.css

```diff
@@ -395,5 +397,11 @@ textarea.Field--has-error:focus:hover,
 .cauldron--theme-dark .Radio__overlay--disabled,
 .cauldron--theme-dark .Checkbox__overlay--disabled.Icon--checkbox-checked,
 .cauldron--theme-dark .Checkbox__overlay--disabled.Icon--checkbox-unchecked {
-  color: #4c99a9;
+  color: #5d676f;
+}
+
+.Checkbox__overlay.Checkbox__overlay--focused,
+.Radio__overlay.Radio__overlay--focused {
+  border-radius: 5px;
+  box-shadow: 0 0 0 2px var(--focus-glow);
```

Comment:
It looks like this was defined twice.

Reply:
Ah, this was originally supposed to include `.cauldron--theme-dark`, but forgot to add that, though it looks like the way the CSS variables work, it's not necessary after all. Removed!